cla-backend/cla/models/github_models.py

```diff
@@ -888,46 +888,60 @@ def get_pull_request_commit_authors(pull_request):
     """
     cla.log.debug('Querying pull request commits for author information...')
     commit_authors = []
-    for commit in pull_request.get_commits():
-        cla.log.debug('Processing commit while looking for authors, commit: {}'.format(commit.sha))
+
+    # For each commit associated with this pull request...
+    for commit_model in pull_request.get_commits():
+        cla.log.debug('Processing commit while looking for authors, commit: {}'.format(commit_model.sha))
         # Note: we can get the author info in two different ways:
-        if commit.author is not None:
+        # Note: commit.author is not None doesn't seem to catch empty author child elements
+        if 'author' in commit_model \
```

Comment:
With pygithub=1.5 version the author name check raises a `github.GithubException.IncompletableObject` exception which we can handle. For the use case we fail to get the user we can get the authors details directly from the github-api endpoint and if that fails we pass a None for the author.name detail.